index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="style.css">
+	<script src="https://kit.fontawesome.com/yourcode.js" crossorigin="anonymous"></script>
+	<title>Portfolio</title>
+</head>
+<body>
+	<section class="screen_container">
+		<header class="toolbar_container">
+			<a href="#" class="Title_logo">My Logo</a>
+			<nav>
+				<img src="./images/i.png"  class="logo" alt="Icon menu">
+			</nav>
+		</header>
+
+		<main class="main_container">
+			<section class="text_container">
+				<h1 class="paragraphe">Hey There. <br/>I’m Leader.</h1>
+				
+				<h4>I'm a Software Developer</h4>
+				<p class="para">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+					don’t hesitate to contact me.</p>
+			</section>
+			
+			<section class="icons-container">
+				<ul class="icons-containers">
+					<li><img src="./images/Normal Button/Tertiary/Icons/Icon -GitHub.png"/></li>
+					<li><img src="./images/Icon - Linkedin.png"/></li>
+					<li><img src="./images/Normal Button/Tertiary/Enabled.png"/></li>
+					<li><img src="./images/Normal Button/Tertiary/Icons/Icon - Twitter.png"/></li>
+					<li><img src="./images/Normal Button/Tertiary/Icons/Icon - Medium.png"/></li>	
+				</ul>				
```

Comment:
- [ ] Kindly consider wrapping each and every `<img src="./images/Normal Button/Tertiary/Icons/Icon -GitHub.png"/>` inside a `<a href="">` anchor tag. because apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. it would look like this:
`<li><a href=""> <img src="./images/Normal Button/Tertiary/Icons/Icon -GitHub.png"/> </a> </li>`

Reply:
Fixed.

Reply:
this is fixed